### DISCUSSION TITLE
Run tests on AppVeyor

### DISCUSSION THREAD

**Discussion by Bykiev** [AdamWhiteHat/BigDecimal#39] [Discussions]
I see many tests are not passing on my machine, anybody is running them? It'll be good to run tests on each commit like in other projects

**Comment by Protiguous** [comment]
I am not familiar with AppVeyor. Can you describe what it does?

**Comment by Bykiev** [comment]
You can check this repository: https://github.com/sklose/NCalc2
Every PR has a green/red mark after running the tests on AppVeyor

**Comment by AdamWhiteHat** [comment]
Was the problem with the tests related to the current culture and numbers formatting (which has been fixed now with your changes)?
